content/app/ui/popups/goto-file-popup.js

```diff
@@ -30,15 +39,30 @@ define("content/app/ui/popups/goto-file-popup", function(require, exports, modul
 		 * Init popup.
 		 */
 		_init : function () {
+
+			// Fuzzy set
+			this._fuzzySet = new FuzzySet();
+			this._entries.forEach(function(entry) {
+				this._fuzzySet.add( entry.toString() );
+			}, this);
+
 			// create input
 			this._input = document.createElement('input');
 			this._input.type = 'text';
 			this._input.className = 'popup-goto-file-input';
 			this.view.appendChild(this._input);
+
 			// create label
 			var label = document.createElement('div');
 			label.className = 'popup-goto-file-label';
 			this.view.appendChild(label);
+
+			// init jquery autocomplete widget
+			$(this._input).autocomplete({
```

Comment:
As you introduced jQuery, we should then wrap all DOM elements with it. It's much easier and bug proof.
I suggest that you modify view and _input to keep elements wrapped with jQuery and rename them to _$input and $view. And setting attributes and className can be done using jQuery methods (attr and class/togleClass).
When you merge this branch we should do refactor in all components to use jQuery.